packages/ra-ui-materialui/src/input/AutocompleteInput.tsx

```diff
@@ -496,16 +498,9 @@ If you provided a React element for the optionText prop, you must also provide t
                 onChange={handleAutocompleteChange}
                 onBlur={field.onBlur}
                 onInputChange={handleInputChange}
-                renderOption={(props, record) => {
-                    if (isValidElement(optionText)) {
-                        return cloneElement(optionText, {
-                            record: record as RaRecord,
-                            ...props,
-                        });
-                    }
-
-                    return <li {...props}>{getChoiceText(record)}</li>;
-                }}
+                renderOption={(props, record) => (
```

Comment:
You are not asking for `optionText` to return it

Reply:
I don't understand your question

Reply:
That's what `getChoiceText` does

Reply:
Dah, I of course, my bad!